default/app/app.css

```diff
@@ -1,6 +1,4 @@
-@tailwind base;
-@tailwind components;
-@tailwind utilities;
+@import "tailwindcss";
```

Comment:
@azzzy any reason this `app.css` is missing the `@theme` declaration in all the other templates